sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -20,6 +20,7 @@ import {
 } from '../constants';
 
 const Utils = require('../utils');
+const Log = require('../log');
```

Comment:
I suppose this export is not necessary. What do you thing about deleting this line?

Reply:
Done